next/README.md

```diff
@@ -1,73 +1,54 @@
+| <img src="figures/s2.png" width="600px" alt="S2 Tiling Scheme"> | [`3DTILES_bounding_volume_S2`](../extensions/3DTILES_bounding_volume_S2) (3D Tiles 1.0 extension) <br> Enables the use of [S2 Geometry](http://s2geometry.io/) as a bounding volume and tiling scheme. Particularly when combined with `3DTILES_implicit_tiling`, S2 is well suited for global scale tilesets, minimizing distortion near the poles. Cells at the same level of detail have approximately equal area. |
 
-This extension defines several granularities of metadata:
+In addition to size and performance benefits, these extensions provide improved interoperability with geospatial formats and libraries such as CDB, TMS, WMTS, and S2.
 
-* Tileset Metadata
-* Group Metadata
-* Tile Metadata
+## Metadata
 
-`3DTILES_metadata` also interacts with other extensions to add metadata capability:
+_**Overview:** Metadata in 3D Tiles gains more expressiveness and flexibility, with a well-defined type system, new encoding options (e.g. JSON or binary), and a range of granularity options. Metadata may be associated with high-level objects like tilesets, tiles, or tile content groups, or with individual vertices and texels on glTF 2.0 geometry._
 
-* `3DTILES_implicit_tiling` - tile metadata can be stored in binary subtree files.
-* `3DTILES_multiple_contents` - Each content in a tile can independently be added to groups for group metadata. This allows for more complex grouping behavior, such as organizing contents into layers. It also makes it easier to style and filter content using the group ID.
+The foundation of these new metadata extensions is the format-agnostic [3D Metadata Specification](../specification/Metadata). The specification defines key concepts including schemas, property types, storage formats, and semantic meanings of properties. Extensions for metadata in 3D Tiles and glTF 2.0 are available now, and other formats can be extended to support the common format of the 3D Metadata Specification in the future.
 
-#### [`EXT_mesh_features`](https://github.com/CesiumGS/glTF/tree/3d-tiles-next/extensions/2.0/Vendor/EXT_mesh_features) (glTF 2.0 extension)
+|   | Extension |
+|---|-----------|
+| <img src="figures/metadata-granularity.png" alt="Metadata in 3D Tiles" width="600"> | [`3DTILES_metadata`](../extensions/3DTILES_metadata) (3D Tiles 1.0 extension) <br> Defines 3D Metadata for tilesets, tiles, and tile content groups. Complements other 3D Tiles Next extensions with additional metadata-related features: <br><br>• In `3DTILES_implicit_tiling` tile metadata may be stored compactly in binary subtree files, allowing efficient streaming of large global tilesets and their metadata. <br>• In `3DTILES_multiple_contents` each tile content may be associated with a group and with group metadata, enabling improved content organization, styling, and filtering. |
```

Comment:
Just to check, is the extended metadata diagram needed? if not, would be good to remove it from the PR

Reply:
I'd included it becomes I think it's a helpful overview of how metadata works, and which extension should be used depending on the level of granularity required – could you say more about why it would be good to remove it?

If that's something about the size being too much in this context, we could scale it down:

![Screen Shot 2021-11-09 at 8 55 33 AM](https://user-images.githubusercontent.com/1848368/140970186-86ddf792-b8ee-40e0-8b4b-1116e261e923.png)

Reply:
Oh sorry I somehow missed seeing it referenced in the diff, ignore me